src/Cards/WishCard.js

```diff
@@ -0,0 +1,68 @@
+import "./ProductCard.css";
+import { useMyReducer } from "../stateContext";
+import { BsTrashFill } from "react-icons/bs";
+import { Link } from "react-router-dom";
+import { useLangContext } from "../AllContext/languageContext";
+export const WishCard = (item) => {
+  const {
+    id,
+    imageSrc,
+    productName,
+    description,
+    price,
+    originalPrice,
+    newFlag,
+    quantity
+  } = item;
+  const { dispatch } = useMyReducer();
+  const { language } = useLangContext();
+  const moveToCartFromWishListHandler = (id) => {
+    dispatch({
```

Comment:
moveToCartFromWishList itself depicts the action, you  should remove Handler from function name wherever you might have used

Reply:
Done!